client/src/components/ClueModal.js

```diff
@@ -53,9 +56,38 @@ const ClueModal = (props) => {
         setNumOfGuesses(e.target.value);
     }
 
+    const isOneAlphabeticWord = (word) => {
+        const english = /^[A-Za-z]*$/;
+        return english.test(word);
+    }
+    const isCardWord = (word) => {
+        const cards = props.cards;
+        let i, j;
+        for (i = 0; i < cards.length; i++) {
```

Comment:
this can be rewritten so that it's not so complicated 

1. The incoming parameter `word` can be trimmed at the beginning and saved as a constant and transformed to lowercase
2. If you're looping through all the cards, you can do something like the following in a couple of lines

```
const findCardWord = cards[0].find(card => card.word.toLowerCase() === word)
```

I'm not to sure what `props.cards` looks like, if it's an array of just one element, bceause your second nested for loop only ever accesses the first element of the array